ci/check_diff.sh

```diff
@@ -89,10 +90,18 @@ function check_diff() {
         --unified=0 --no-index rustfmt_diff.txt feature_diff.txt 2>&1 | tail -n +6 | cut -c 2-
     )
 
+    # COPY diffs into output dir
+    mkdir $OUTPUT_DIR/$1
+    echo "Copying diffs to $OUTPUT_DIR/$1"
+    cp rustfmt_diff.txt $OUTPUT_DIR/$1/rustfmt_diff.txt
+    cp feature_diff.txt $OUTPUT_DIR/$1/feature_diff.txt
```

Comment:
And just another nit, but we could similarly create a variable for the target output directory and reuse it instead of having multiple directory concat instances